ios/MagnetScannerReact.swift

```diff
@@ -14,12 +17,41 @@ import Foundation
   }
   
   @objc func stop() {
-    scanner.stop()
+    // We don't stop scanning in iOS, we let the OS to
+    // throttle the notifications of beacons found.
+    // scanner.stop()
   }
   
   func onItemFound(item: Dictionary<String, AnyObject>) {
-    print("item found", item["url"]);
+    print("item found", item["url"])
     bridge.eventDispatcher()
-      .sendDeviceEventWithName("magnetscanner:itemfound", body: item);
+      .sendDeviceEventWithName("magnetscanner:itemfound", body: item)
+    
+    notifyUser(item["url"] as! String)
+  }
+  
+  // Throttles the notification process, waiting for 1 minute until
+  // setting up the badge with the number of elements nearby.
+  func notifyUser(url: String) {
+    if (toNotify.contains(url)) {
+      return;
+    }
```

Comment:
Actually no, a Dictionary is an structure that we don't need here, as it's a <key,value>, and we don't need to store anything as value.
Also  the arrays in swift have function like contains that is enough to tell us if we already have an element on it.